commit 0ea1d601ee56e1eb81be5f0a740e1380735e11ac
Author: Andreas Boll <aboll@debian.org>
Date:   Mon Feb 11 14:48:08 2019 +0100

    Revert "Map evdev keycode KEY_KEYBOARD to XF86Keyboard"
    
    This reverts commit d0e5e63cf59be444bec0e5b11838a9abce45226d.
    
    This patch breaks keyboard layouts in XWayland applications when having several
    keyboard layouts enabled.
    
    See also https://bugzilla.redhat.com/show_bug.cgi?id=1587998
    
    Fixes: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=921867
    Fixes: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=921940
    Fixes: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=922020
    
    Drop this revert once we have a newer xkbcomp (>> 1.4.2) and bump
    dependency on x11-xkb-utils.

diff --git a/keycodes/evdev b/keycodes/evdev
index 90b8278d..f2630ffe 100644
--- a/keycodes/evdev
+++ b/keycodes/evdev
@@ -285,7 +285,6 @@ default xkb_keycodes "evdev" {
 	<I255> = 255;	// #define KEY_RFKILL              247
 
 	<I372> = 372;   // #define KEY_FAVORITES           364
-	<I374> = 374;   // #define KEY_KEYBOARD            366
 
 	// Fake keycodes for virtual keys
 	<LVL3> =   92;
diff --git a/symbols/inet b/symbols/inet
index 4d774fac..469a7d65 100644
--- a/symbols/inet
+++ b/symbols/inet
@@ -219,7 +219,6 @@ xkb_symbols "evdev" {
     key <I255>   {      [ XF86RFKill            ]       }; // KEY_RFKILL
 
     key <I372>  {       [ XF86Favorites          ]       };
-    key <I374>  {       [ XF86Keyboard           ]       };
 
     key <FK13>   {      [ XF86Tools         ]       };
     key <FK14>   {      [ XF86Launch5       ]       };
